doc/guide/nix.md

```diff
@@ -1,11 +1,43 @@
 # Using Gerbil with Nix
 
-You can use Gerbil with the Nix package manager, and it will guarantee you that Gerbil is correctly built, in a deterministic way, on top of Gambit properly built with all the correct options.
+You can use Gerbil with the [Nix package manager](https://nixos.org), and
+it will guarantee you that Gerbil is correctly built, in a deterministic way,
+on top of an appropriate Gambit properly built with all the correct options.
```

Comment:
what gambit? It's a submodule.

Reply:
Not on NixOS, because you don't have access to git during the build, and it's easier to ride on the existing gambit support and splice a gambit tree from nix (stable or unstable) into the gerbil tree.

Reply:
uhm ok